Migration can be unpermitted (#4142)
quicwg/base-drafts: draft-ietf-quic-transport.md

Receiving a packet from a new peer address containing a non-probing frame
indicates that the peer has migrated to that address.

> If the recipient permits the migration, an endpoint MUST send subsequent packets

Reviewer: ```suggestion
If the recipient permits the migration, it MUST send subsequent packets
```
